diff -up ibacm-1.0.5/src/acm.c.conf ibacm-1.0.5/src/acm.c
--- ibacm-1.0.5/src/acm.c.conf	2012-02-28 13:50:38.316133667 -0500
+++ ibacm-1.0.5/src/acm.c	2012-02-28 13:53:47.278117602 -0500
@@ -205,8 +205,8 @@ static atomic_t counter[ACM_MAX_COUNTER]
 /*
  * Service options - may be set through acm_opts file.
  */
-static char *opts_file = "/etc/ibacm/acm_opts.cfg";
-static char *addr_file = "/etc/ibacm/acm_addr.cfg";
+static char *opts_file = "/etc/rdma/acm_opts.cfg";
+static char *addr_file = "/etc/rdma/acm_addr.cfg";
 static char log_file[128] = "/var/log/ibacm.log";
 static int log_level = 0;
 static char lock_file[128] = "/var/lock/ibacm.pid";
